core/utils/dmljob.py

```diff
@@ -47,31 +47,28 @@ def __init__(
                                     `train.csv` and `test.csv` are in folder.
             datapoint_count (int): Number of datapoints after applying
                                    transform function.
-
+            new_weights (list): list of np.arrays representing the weights that
+                                should be averaged with 'weights'
+            omega (float): the weight given to new_weights in weighted averaging
+            sigma_omega (float): the sum of omega's incorporated into weighted
+                                averaging so far. Kept in the DMLJob to allow the
+                                weighted average to be unrolled.
         """
         self.job_type = job_type
         self.serialized_model = serialized_model
         self.framework_type = framework_type
         self.weights = weights
         self.hyperparams = hyperparams
         self.label_column_name = label_column_name
+        self.omega = None
+        self.sigma_omega = None
+        self.new_weights = None
```

Comment:
Any reason why you chose to always set these variables to `None` instead of having them set to `None` as a default option (like other attributes)?

Reply:
Yes. Omega, sigma_omega, and new_weights can only be set after the initialization of a job.